tools/tpm2_activatecredential.c

```diff
@@ -24,30 +25,26 @@
 
 typedef struct tpm_activatecred_ctx tpm_activatecred_ctx;
 struct tpm_activatecred_ctx {
-
-    struct {
-        UINT8 i : 1;
-        UINT8 o : 1;
-    } flags;
-
     struct {
-        char *auth_str;
-        tpm2_session *session;
-        const char *ctx_arg;
-    } key;
+        const char *ctx_path;
+        const char *auth_str;
+        tpm2_loaded_object object;
+    } keyhandle; //Typically EK
```

Comment:
this more than just the keyhandle, isn't it the key itself?

Reply:
The idea was to rename the variables verbatim as in the spec. Yes it is the key itself. Should it instead be called "keyhandle_obj" and "activatehandle_obj"?

Reply:
If the spec isn't descriptive or doesn't add clarity, then I would stay away from following it.

Reply:
SGTM and so then I will rename it to:
activatehandle --> credentialed_obj
keyhandle --> credential_key
I will update the options and man as well.